packages/ckeditor5-dev-env/lib/release-tools/utils/cli.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const inquirer = require( 'inquirer' );
+const semver = require( 'semver' );
+
+const cli = {
+	/**
+	 * Asks a user for a confirmation for creating the releases.
+	 *
+	 * @param {Map} packages Packages to release.
+	 * @returns {Promise}
+	 */
+	confirmRelease( packages ) {
+		let message = 'Packages to release:\n';
+
+		for ( const [ packageName, packageDetails ] of packages ) {
+			message += `  - "${ packageName }": v${ packageDetails.version }\n`;
```

Comment:
You need to remember about consistency. Aren't log entries listed as ` * <TEXT>`? Use the same indentation and list bullet all across the code.